.env.dev.sample

```diff
@@ -1,10 +0,0 @@
-DATABASE_URL=postgres://rcjaregistration:abc123@db/registration
-DEV_SETTINGS=true
-DEBUG=true
-DJANGO_LOG_LEVEL=DEBUG
-SECRET_KEY=TESTONLY_slfgdjheaklfgjb34wuhgb35789gbne97urgblskdg # Any random string
-
-# For database, if running locally
-POSTGRES_DB=registration
-POSTGRES_USER=rcjaregistration
-POSTGRES_PASSWORD=abc123

```

Comment:
@evanbailey I expect this was accidentally deleted... restore also revert change in `.gitignore`